src/misc/8.md

```diff
@@ -0,0 +1,42 @@
+# Misc 8 @adotinthevoid
+
+{{#include ../include/quiz-is-wip.md}}
+
+```rust
+{{#include ../../code/examples/misc_8.rs:matters}}
+```
+
+<details>
+<summary>Solution</summary>
+
+```
+{{#include ../../code/examples/stderr/misc_8.stderr}}
+```
+
+There are 3 kinds of structs in rust:
+
+1. Plain structs (eg `struct Foo{bar: i32}`)
+2. Tuple structs (eg `struct Bar(i32);`)
+3. Unit structs (eg `struct Baz;`)
+
+However all 3 kinds can have a struct with no fields.
+
+Types and values live in seperate namespaces, as it is usually [^not1] possible to
+syntacticly determine weather an identifier the compiller needs to resolve will
+be an value or type .
+
+[^not1]: except when it isn't :)
+
+This means that `struct bar` only gets inserted into the type namespace, and `fn
```

Comment:
```suggestion
This means that `struct bar {}` only gets inserted into the type namespace, and `fn
```
I was confused for a moment why a unit struct would only be in the type namespace, so let's add braces to make it clearer